### PR TITLE
Feature: Add autoscaling profiles

### LINKED ISSUE #455: Add support for autoscaling_profile
Add support for autoscaling_profile in the cluster_autoscaling block. 
https://www.terraform.io/docs/providers/google/r/container_cluster.html#cluster_autoscaling

### PR DESCRIPTION
**Pull request by bharathkkb**
Fixes #455 
- Adds autoscaling_profile to the cluster_autoscaling obj
- Added test

### CODE REVIEW

File: autogen/main/cluster.tf.tmpl [hunk 1]
```diff
@@ -68,7 +68,8 @@ resource "google_container_cluster" "primary" {
 
 {% if beta_cluster %}
   cluster_autoscaling {
-    enabled = var.cluster_autoscaling.enabled
+    enabled             = var.cluster_autoscaling.enabled
+    autoscaling_profile = var.cluster_autoscaling.autoscaling_profile
```
Comment (morgante) [inline autogen/main/cluster.tf.tmpl]: Should we set a default here (BALANCED)?

Comment (bharathkkb) [reply autogen/main/cluster.tf.tmpl]: I have the default in the cluster_autoscaling to BALANCED.

Comment (morgante) [reply autogen/main/cluster.tf.tmpl]: Right, but that default will only be applied if `cluster_autoscaling` is omitted entirely. I should be able to specify `autoscaling_profile = null` to get a sane default (or should the default be disabled?).

Comment (bharathkkb) [reply autogen/main/cluster.tf.tmpl]: ah, yes good point; will update :)

**Pull request merged**